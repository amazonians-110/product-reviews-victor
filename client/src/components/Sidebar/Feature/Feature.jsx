import React from 'react';

class Feature extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return(
      <div>
        <b>By feature</b>
        <div>Sound quality          ***** 4.4</div>
        <div>Battery life           ***** 4.1</div>
        <div>Noise cancellation     ***** 4.1</div>
      </div>
    )
  }
}

export default Feature;