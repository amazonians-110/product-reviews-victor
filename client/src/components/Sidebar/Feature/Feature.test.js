/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import Feature from './Feature';

describe('Feature', () => {
  it('should render correctly', () => {
    const component = shallow(<Feature />);

    expect(component).toMatchSnapshot();
  });
});
