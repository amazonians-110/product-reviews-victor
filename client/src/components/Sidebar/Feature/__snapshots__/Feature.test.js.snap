// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature should render correctly 1`] = `
<styled.div>
  <b>
    By feature
  </b>
  <styled.div>
    Sound quality
    <img
      alt="stars"
      height="25px"
      src="https://s3.amazonaws.com/product-reviews-hr110/Icons/stars.png"
    />
    4.4
  </styled.div>
  <styled.div>
    Battery life
    <img
      alt="stars"
      height="25px"
      src="https://s3.amazonaws.com/product-reviews-hr110/Icons/stars.png"
    />
    4.1
  </styled.div>
  <styled.div>
    Noise cancellation
    <img
      alt="stars"
      height="25px"
      src="https://s3.amazonaws.com/product-reviews-hr110/Icons/stars.png"
    />
    4.1
  </styled.div>
</styled.div>
`;
